serverless/packages/functions/src/model/create_model.ts

```diff
@@ -0,0 +1,51 @@
+import { APIGatewayProxyEventV2 } from "aws-lambda";
+import parseJwt from "@dlp-sst-app/core/src/parseJwt";
+import { v4 as uuidv4 } from 'uuid';
+import { DynamoDBClient, PutItemCommand } from '@aws-sdk/client-dynamodb';
+
+export async function handler(event : APIGatewayProxyEventV2) {
+    if (event) {
+        const user_id: string = parseJwt(event.headers.authorization ?? "")["user_id"];
+        const model_id = uuidv4();
+
+        const client = new DynamoDBClient({});
+
+        const eventBody = JSON.parse(event.body? event.body : "");
+        const putCommand: PutItemCommand = new PutItemCommand({
+            TableName: "ModelTable",
+            Item:
+            {
+                user_id: {"S": user_id},
+                model_id: {"S": model_id},
+                name: {"S": eventBody['name']},
+                model_structure: {"S": eventBody['model_structure']}
```

Comment:
did we ever discuss how model_structure would be populated in dynamo?

@noah-iversen @codingwithsurya @alantao912